### PR TITLE
mgr/dashboard: sync policy's in Object >> Multi-site >> Sync-policy does not show the zonegroup to which policy belongs to  

### PR DESCRIPTION
**Pull request by nmunet**
Fixes: https://tracker.ceph.com/issues/68355

Fixes Includes: Added default zonegroup name with the sync policy details

**BEFORE:**
image

**AFTER:**
image

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [x] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [x] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/controllers/rgw.py [hunk 1]
```diff
@@ -176,13 +176,18 @@ def get_sync_policy(self, bucket_name='', zonegroup_name='', all_policy=None):
         if all_policy:
             sync_policy_list = []
             buckets = json.loads(RgwBucket().list(stats=False))
+            default_zonegroup = ''
+            for zonegroup in RgwMultisite().get_all_zonegroups_info()['zonegroups']:
+                if zonegroup['id'] == RgwMultisite().get_all_zonegroups_info()['default_zonegroup']:
+                    default_zonegroup = zonegroup['name']
```
Comment (afreen23) [inline src/pybind/mgr/dashboard/controllers/rgw.py:182]: Just a concern, are we ensured that these lookups wont results in `KeyError`

**Comment by afreen23** [comment]
jenkins test dashboard

File: src/pybind/mgr/dashboard/controllers/rgw.py (hunk 1 above)
Comment (nmunet) [reply src/pybind/mgr/dashboard/controllers/rgw.py]: added more checks

[the remark this answers is not in the document]
Comment (nmunet) [reply src/pybind/mgr/dashboard/controllers/rgw.py]: Done

**Comment by nmunet** [comment]
jenkins test api

**Comment by nmunet** [comment]
jenkins test dashboard

**Comment by nmunet** [comment]
jenkins test dashboard cephadm

**Comment by nmunet** [comment]
jenkins test make check

**Comment by nmunet** [comment]
jenkins test make check

**Review by nizamial09 — APPROVED** [review]

**Comment by nmunet** [comment]
dashboard & cephadm tests failure seems unrelated to this PR

**Pull request merged**